restock.lic

```diff
@@ -0,0 +1,166 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#restock
+=end
+
+custom_require.call(%w[common common-items common-money])
+
+$NUM_MAP = {
+    'zero'=> 0,
+    'one' => 1,
+    'two'=> 2,
+    'three'=> 3,
+    'four'=> 4,
+    'five'=> 5,
+    'six'=> 6,
+    'seven'=> 7,
+    'eight'=> 8,
+    'nine'=> 9,
+    'ten'=> 10,
+    'eleven'=> 11,
+    'twelve'=> 12,
+    'thirteen'=> 13,
+    'fourteen'=> 14,
+    'fifteen'=> 15,
+    'sixteen'=> 16,
+    'seventeen'=> 17,
+    'eighteen'=> 18,
+    'nineteen'=> 19,
+    'twenty'=> 20,
+    'thirty'=> 30,
+    'forty'=> 40,
+    'fifty'=> 50,
+    'sixty'=> 60,
+    'seventy'=> 70,
+    'eighty'=> 80,
+    'ninety'=> 90
+}
+
+class Restock
+  include DRC
+
+  def initialize
+    setup
+
+    items = parse_restockable_items
+    items_to_restock = Array.new
+    coin_needed = 0
+
+    items.each do |item|
+      if item['stackable']
+        remaining = count_stackable_item(item['name'])
+      else
+        remaining = count_nonstackable_item(item['name'])
+      end
+ 
+      if remaining < item['quantity']
+        num_needed = item['quantity'] - remaining
+        buy_num = (num_needed / item['size'].to_f).ceil
+        coin_needed += buy_num * item['price']
+        item['buy_num'] = buy_num
+        items_to_restock.push(item)
+      end
+    end  
+
+    DRCM.ensure_copper_on_hand(coin_needed, @hometown) if coin_needed > 0
```

Comment:
What do you think about adding `include DRCM` and `include DRCI` around line 39?

Reply:
done!